File: Java/Practice/Fibonacci.java
package Practice;
import javax.swing.JOptionPane;
public class Fibonacci {
    public static void main(String[] args){
       int num1 = 0, num2 = 1;
       String len = JOptionPane.showInputDialog("Enter how many digits of the Fibonacci sequence you want: ");
       int length = Integer.parseInt(len);
       for(int i = 1; i <= length; i++){
        System.out.print(num1 + " ");
        int num3 = num1 + num2;
        num1 = num2;
        num2 = num3;
       }
    }
}

File: Python/fibonacci.py
num1 = 0
num2 = 1
inp = int(input("Enter how many digits of the Fibonacci Sequence you want: "))
for i in range(inp):
    print(num1, end=" ")
    num3 = num1 + num2
    num1 = num2
    num2 = num3